New pages for HTML*Element.form (#35661)
mdn/content: files/en-us/web/api/htmllabelelement/form/index.md

{{APIRef("HTML DOM")}}

> The **`form`** read-only property of the {{domxref("HTMLLabelElement")}} interface returns an {{domxref("HTMLFormElement")}} object that owns the {{domxref("HTMLLabelElement.control", "control")}} associated with this label, or `null` if this label is not associated with a control owned by a form.

Reviewer: ```suggestion
The **`form`** read-only property of the {{domxref("HTMLLabelElement")}} interface returns the {{domxref("HTMLFormElement")}} object that owns the form {{domxref("HTMLLabelElement.control", "control")}} the {{HTMLElement("label")}} is associated with, or `null` if the label is not associated with a control owned by a form.
```

Reviewer: do we want to mention the `form` attribute may be set on the associated form control, but not the label?
